tests/unit/common/test_multi.py

```diff
@@ -0,0 +1,55 @@
+from hdmf.common import DynamicTable, VectorData, \
+    DynamicTableRegion, SimpleMultiContainer
+from hdmf.testing import TestCase, H5RoundTripMixin
+
+
+class DynamicTableRegionRoundTrip(H5RoundTripMixin, TestCase):
```

Comment:
I think the test for SimpleMultiContainer should not be dependent on DynamicTable and DynamicTableRegion working. It could be done with generic Container and Data objects.